scripts/populate_institutions.py

```diff
@@ -91,6 +91,16 @@ def main(env):
                 'email_domains': [],
             },
             {
+                '_id': 'nyu',
```

Comment:
Not sure how these unrelated changes made it in. Was this PR branched off of master perchance?